OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -535,6 +545,23 @@ void ITick.Tick(Actor self)
 
 			foreach (var b in builders)
 				b.Tick();
+
+			Sync.CheckSyncUnchanged(World, () =>
```

Comment:
While we're here it would be good for debugging if we could wrap this in a new "bot_tick" perf sampler